reco_utils/dataset/wikidata.py

```diff
@@ -152,9 +153,8 @@ def read_linked_entities(data):
         data (json): dictionary with linked pages
 
     Returns:
-        list, list: 
-        - List of liked entityIDs
-        - List of liked entity names
+        (list): List of liked entityIDs
+        (list): List of liked entity names
```

Comment:
what is the effect of having the parenthesis here instead of the previous approach?